terraform/main.go

```diff
@@ -0,0 +1,60 @@
+package main
+
+import (
+	"encoding/json"
+	"os"
+
+	helper "goterra/pkg/helper"
+	ihp "goterra/pkg/incompetentHostingProvider"
+	logging "goterra/pkg/logging"
+
+	log "github.com/rs/zerolog/log"
+)
+
+var cwd, _ = os.Getwd()
+var tf_bin_dir string = cwd + "/bins"
+var tf_version string = "1.7.1"
```

Comment:
const

Reply:
Changed the version to be a constant in last commit. Other variables can't be declared as const since the value is unknown before runtime.